### PR TITLE
Fix/api securityschemes

### LINKED ISSUE #3489: Add Authentication (securitySchemes) to v1.x API contracts and docs
V1Compat openapi contracts are missing authentication (securityScheme) sections.

##### Relevant URLs
- _Provide relevant URLs (the doc page in question, project issues, community threads, etc.)_

### LINKED ISSUE #3490: Remove Querystring and Basic auth (securitySchemes) from /api/v2 docs
/api/v2 OSS and Cloud openapi contracts and API docs should not include Querystring and Basic auth (securitySchemes). Those schemes should only apply to v1compat API.

##### Relevant URLs
- _Provide relevant URLs (the doc page in question, project issues, community threads, etc.)_

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #3489
Closes #3490

Add an openapi CLI decorator that replaces INFLUXDB_DOCS_URL 'shortcodes' with the correct URL in specs.

Fix API authentication descriptions and schemes in /api/v2 and V1Compat.
Remove QueryString and Basic auth schemes from /api/v2 and add them to V1Compat, along with Token auth.
Hook up the replace-docs-url-shortcode which is primarily for V1Compat for now due to the way that contract is managed.
Add tag groups to V1Compat.
Document and comment out Bearer auth in OSS until it's available in Cloud.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by jstirnaman** [comment]
Still ignoring lots of grammar problems for now.

File: api-docs/cloud/ref.yml [hunk 1]
```diff
@@ -12287,19 +12220,10 @@ paths:
...
-                  value: >
-                    {"code":"request too large","message":"unable to read data:
-                    points batch is too large"}
-              schema:
-                $ref: '#/components/schemas/LineProtocolLengthError'
             text/html:
               examples:
                 dataExceedsSizeLimit:
-                  summary: InfluxDB Cloud response
+                  summary: Cloud response
```
Comment (sanderson) [inline api-docs/cloud/ref.yml:12226]: Any specific reason for removing "InfluxDB" here?

File: api-docs/cloud/swaggerV1Compat.yml [hunk 2]
```diff
@@ -1,14 +1,20 @@
-# this is a manually maintained file for these old routes until oats#15 is resolved
-openapi: "3.0.0"
+openapi: 3.0.0
 info:
   title: Influx API Service (V1 compatible endpoints)
   version: 0.1.0
+  description: |
+    The InfluxDB 1.x compatibility /write and /query endpoints work with
```
Comment (sanderson) [inline api-docs/cloud/swaggerV1Compat.yml:6]:
```suggestion
    The InfluxDB 1.x compatibility `/write` and `/query` endpoints work with
```

File: api-docs/cloud/swaggerV1Compat.yml [hunk 3]
```diff
@@ -21,15 +27,17 @@ paths:
...
+        - $ref: '#/components/parameters/AuthUserV1'
+        - $ref: '#/components/parameters/AuthPassV1'
         - in: query
           name: db
           schema:
             type: string
           required: true
-          description: The bucket to write to. If none exist a bucket will be created with a default 3 day retention policy.
+          description: >-
+            The bucket to write to. If none exist a bucket will be created with
```
Comment (sanderson) [inline api-docs/cloud/swaggerV1Compat.yml:39]:
```suggestion
            Bucket to write to. If none exist a bucket will be created with
```

File: api-docs/cloud/swaggerV1Compat.yml [hunk 4]
```diff
@@ -142,48 +180,70 @@ paths:
             type: string
             enum:
               - application/vnd.influxql
+        - in: query
+          name: db
+          schema:
+            type: string
+          required: true
+          description: The bucket to query.
```
Comment (sanderson) [inline api-docs/cloud/swaggerV1Compat.yml:188]:
```suggestion
          description: Bucket to query.
```

**Review by sanderson — APPROVED** [review]
Some minor suggests, but then it's gtg.

**Pull request merged**